### PR TITLE
Modify distances acoording to dpi

### PR DESCRIPTION
**Pull request by oterral**
Fix https://github.com/geoadmin/mf-geoadmin3/issues/3276

[Test](https://mf-geoadmin3.dev.bgdi.ch/teo_printwidth/?lang=fr&topic=ech&bgLayer=ch.swisstopo.pixelkarte-farbe&X=252454.82&Y=747104.44&dev3d=true&zoom=7&debug&layers=ch.swisstopo.zeitreihen,ch.bfs.gebaeude_wohnungs_register,ch.bafu.wrz-wildruhezonen_portal,ch.swisstopo.swisstlm3d-wanderwege,KML%7C%7Chttps:%2F%2Fpublic.dev.bgdi.ch%2Fceray2cFSBuuwT4I-_QYLQ&layers_visibility=false,false,false,false,true&layers_timestamp=18641231,,,,)

@mariokeusen can you test please ?

### CODE REVIEW

File: src/components/print/PrintDirective.js [hunk 1]
```diff
@@ -951,6 +951,14 @@ goog.require('ga_time_service');
       }
     };
 
+    // Change a distance according to the change of DPI
+    var adjustDist = function(dist) {
+      if (!dist) {
+        return;
+      }
+      return dist * 90 / getDpi($scope.layout.name, $scope.dpi);
```
Comment (gjn) [inline src/components/print/PrintDirective.js]: What is this magic number?

Comment (oterral) [inline src/components/print/PrintDirective.js]: The OGC standard output resolution is 90 DPI, it's used also by openlayers.

Comment (loicgasser) [inline src/components/print/PrintDirective.js]: 90.7 to be accurate ;)

Comment (gjn) [inline src/components/print/PrintDirective.js]: 90.7 is DPI in WMTS standard (used to calcualte scalefactor in WMTS GetCap).

Our WMTS Tiles are calculated with 96DPI and our WMS uses 96dpi as standard output. In print, we are using 150DPI.

Also, we use 72 DPI insided this print directive, I think to create the preview rectangle.

If I'm not completely mistaken, we should use 96 in this function here...or 72.

**Pull request merged**